chrome.js

```diff
@@ -64,10 +65,27 @@ class ChromeWebDriverInstaller extends WebDriverInstallerBase {
    */
   async getBestDriverVersion(browserVersion) {
     const idealMajorVersion = parseInt(browserVersion.split('.')[0], 10);
-    return await InstallerUtils.fetchVersionUrlWithAutomaticDowngrade(
-        idealMajorVersion,
-        /* minMajorVersion */ idealMajorVersion - 2,
-        (majorVersion) => `${CDN_URL}/LATEST_RELEASE_${majorVersion}`);
+
+    // Chrome distribution changed in version 115.
+    if (idealMajorVersion < 115) {
+      return await InstallerUtils.fetchVersionUrl(
+          `${CDN_URL}/LATEST_RELEASE_${idealMajorVersion}`);
+    } else {
+      const data = await this.getVersionData_();
+
+      // Fall back on the major version if necessary.
+      // We haven't seen this become necessary yet since the new distribution
+      // mechanism debuted, but better safe than sorry.
+      let majorVersion = idealMajorVersion;
+      while (majorVersion >= 115) {
```

Comment:
I see the number 115 is repeated in the code, better to extract it to a constant.

Reply:
Done